src/Analyzers/Core/Analyzers/RemoveUnusedMembers/AbstractRemoveUnusedMembersDiagnosticAnalyzer.cs

```diff
@@ -725,7 +783,7 @@ private bool IsCandidateSymbol(ISymbol memberSymbol)
                                     }
 
                                     // Ignore methods which make a type awaitable.
-                                    if (methodSymbol.ContainingType.AllInterfaces.Contains(_iNotifyCompletionType, SymbolEqualityComparer.Default)
+                                    if (_iNotifyCompletionType != null && Roslyn.Utilities.ImmutableArrayExtensions.Contains(methodSymbol.ContainingType.AllInterfaces, _iNotifyCompletionType, SymbolEqualityComparer.Default)
                                         && methodSymbol.Name is "GetAwaiter" or "GetResult")
```

Comment:
>methodSymbol.Name is "GetAwaiter" or "GetResult" [](http://example.com/codeflow?start=43&length=48)

I don't have much context here, but might it be better to check the symbolname before searching the containing types interfaces?